apps/vh/test-api/stg.yaml

```diff
@@ -9,5 +9,4 @@ spec:
     java:
       ingressHost: vh-test-api.staging.platform.hmcts.net
       environment:
```

Comment:
this is missing a lot of environment variable overrides.

Reply:
will complete in new PR